src/ocean/meta/types/Arrays.d

```diff
@@ -107,3 +107,30 @@ unittest
     static assert (is(StripAllArrays!(int[float][4][][]) == int[float]));
     static assert (is(StripAllArrays!(double) == double));
 }
+
+/*******************************************************************************
+
+    Params:
+        T = any type
+
+    Returns:
+        If T is a static array, evaluates to matching dynamic array. Othwerwise
+        evaluates to just T.
+
+*******************************************************************************/
+
+public template SliceIfStaticArray ( T )
```

Comment:
That looks... Utterly specific

Reply:
I was very surprised to see how widespread it is in ocean. Not being able to return static arrays from functions makes some far reaching impact on generic code.